webextension/privileged/prefs/prefs.jsm

```diff
@@ -0,0 +1,51 @@
+"use strict";
```

Comment:
This should not be a jsm, it needs to be a simple script that defines a subclass of `ExtensionAPI` called `prefs`.  See https://firefox-source-docs.mozilla.org/toolkit/components/extensions/webextensions/functions.html#implementing-a-function-in-the-main-process